.github/workflows/release-rai.yml

```diff
@@ -60,18 +60,11 @@ jobs:
         run: |
           yarn buildall
 
-      - if: ${{ matrix.operatingSystem != 'macos-latest' }}
-        name: Install pytorch on non-MacOS
+      - name: Install pytorch
         shell: bash -l {0}
         run: |
           conda install --yes --quiet pytorch torchvision captum cpuonly -c pytorch
 
-      - if: ${{ matrix.operatingSystem == 'macos-latest' }}
-        name: Install Anaconda packages on MacOS, which should not include cpuonly according to official docs
-        shell: bash -l {0}
-        run: |
-          conda install --yes --quiet pytorch torchvision captum -c pytorch
```

Comment:
we don't need this anymore?

Reply:
yes we still install pytorch but for ubuntu only since release script only runs in one OS, this statement was just not run